app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,15 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                ng-attr-aria-disabled="{{row.canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !row.canDeploy() }">Deploy (paused)</a>
```

Comment:
I can wrap the `(paused)` with a `span` and `ng-if="isPaused()"` 

Also... should the above `switch` be updated? I was considering updating the `switch` to `DeploymentConfig || Deployment`, but not sure if that is right. Do we want a `Resume Rollouts` button added here?  I initially assumed we should keep this menu slim, but seems like that might actually be pretty helpful in this case.